### ISSUE TITLE
Generate code for body-byte.json

### ISSUE THREAD

**Issue by jianghaolu** [Azure/autorest.java#410]

**Comment by weidongxu-microsoft** [comment]
Swagger response of type `byte array`

```
        "/byte/null": {
            "get": {
                "operationId": "byte_getNull",
                "description": "Get null byte value",
                "tags": [
                    "Byte Operations"
                ],
                "responses": {
                    "200": {
                        "description": "The null byte value",
                        "schema": {
                            "description": "The null byte value",
                            "type": "string",
                            "format":  "byte",
                            "enum": [null]
                        }
                    },
                    "default": {
                        "description": "Unexpected error",
                        "schema": {
                            "$ref": "#/definitions/Error"
                        }
                    }
                }
            }
```

got transferred to `string `by autorest.core

```
  constants:
  - &ref_4
    type: constant
    apiVersions:
    - version: 1.0.0
    value:
      language:
        default:
          name: ''
          description: ''
    valueType: &ref_0
      type: string
      language:
        default:
          name: string
          description: simple string
      protocol: {}
    language:
      default:
        name: Constant0
        description: The null byte value
    protocol: {}
```

Hence we have `String` in response instead of `byte[]`.

```
    @ServiceMethod(returns = ReturnType.SINGLE)
    public Mono<SimpleResponse<String>> getNullWithResponseAsync() {
        return service.getNull();
    }
```

Is this expected?

**Comment by weidongxu-microsoft** [comment]
Similar for body-datetime https://github.com/Azure/autorest.java/issues/414
(seems the `enum` in swagger is trggering this)

```
  constants:
  - &ref_19
    type: constant
    apiVersions:
    - version: 1.0.0
    value:
      value: '0001-01-01T00:00:00Z'
      language:
        default:
          name: ''
          description: ''
    valueType: &ref_0
      type: string
      language:
        default:
          name: string
          description: simple string
      protocol: {}
    language:
      default:
        name: Constant0
        description: MISSING·SCHEMA-DESCRIPTION-CHOICE
    protocol: {}
```

**Comment by weidongxu-microsoft** [comment]
Other json having single enum would be:

```
header.json
httpInfrastructure.json
url
```

**Comment by weidongxu-microsoft** [comment]
https://github.com/Azure/autorest.modelerfour/issues/55

**Comment by jianghaolu** [comment]
fixed via #458
